tests/monitor/test_monitor_resource.py

```
@@ -2,112 +2,148 @@
+                    MagicMock(read_count=60, write_count=25, read_bytes=2048 + 51200, write_bytes=1024 + 25600),
+                ]
 
-            # Metrics should still be populated
-            metrics = monitor.metrics
-            expected_peak_memory = 0.5
-            assert "peak_memory_gb" in metrics
-            assert metrics["peak_memory_gb"] == expected_peak_memory  # 512 MB in GB
+                def code_that_raises():
+                    with monitor.measure():
+                        time.sleep(0.25)
+                        raise ValueError("Test exception")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)